doc/guides/security-release-process.md

```diff
@@ -35,6 +35,8 @@ information described.
 
 ## Announcement (one week in advance of the planned release)
 
+* [ ] Verify that Github Actions are working as normal: https://www.githubstatus.com/.
```

Comment:
Should this be in a different section? Seems like verifying this would be more of a last minute thing, not a "on week in advance" thing?

Reply:
This is actually the _last minute_ thing. We can't publish to the website if the Github Actions are down.